app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str):
+    check_list = command.split(" ")
```

Comment:
split automatically splits on spaces, no need to use " "

Reply:
Thank you for useful advice